src/Ledger/Chain.lagda

```diff
@@ -3,31 +3,26 @@
 \begin{code}[hide]
 {-# OPTIONS --safe #-}
 
-open import Ledger.Transaction
-
-module Ledger.Chain (txs : TransactionStructure) where
-
-open import Ledger.Prelude
-
-open TransactionStructure txs
-
 open import Algebra
 open import Data.Maybe using (_>>=_)
 open import Data.Nat.Properties using (+-0-monoid; +-0-commutativeMonoid)
-open import Ledger.Gov TxId Network ADHash epochStructure ppUpd ppHashingScheme crypto
-open import Ledger.Ledger txs
-open import Ledger.PParams epochStructure
-open import Ledger.Ratify txs
-open import Ledger.Utxo txs
 
-open Equivalence
+open import Ledger.Prelude; open Equivalence
+open import Ledger.Transaction
+
+module Ledger.Chain (⋯ : _) (open TransactionStructure ⋯) where
```

Comment:
My rationale is that (even a non-expert) would only need the information of *what kind of parameters* do we have here, in this case `TransactionStructure`. Putting `txs` there for the variable (or even duplicating the type `TransactionStructure` god forbid) would include unnecessary information, which would confuse non-Agda readers, but might be less confusing for Agda newbies instead coming from pure-ASCII languages like Haskell. 

So, there is a clashing constraint here, let me know what you prefer.

Reply:
Even knowing better, to me `(⋯ : _)` looks like special syntax such as `⦃...⦄`. I don't actually mind duplication, since it's equivalent anyway, and any change to one will immediately cause a type error in the other.

Reply:
I am confused, do you prefer to avoid `_` and duplicate the type `TransactionStructure`? Or change the unicode dots to `txs`? Or both?

Reply:
I think any of those options is fine. The one with the least amount of overhead would of course be changing the dots to `txs`.